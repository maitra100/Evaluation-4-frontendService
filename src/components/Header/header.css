#headertext{
    margin-left: 7%;
    font-size: 1.5vw;
    margin-top: 12px;
}

#header{
    height:55px;
    background-color: aliceblue;
}