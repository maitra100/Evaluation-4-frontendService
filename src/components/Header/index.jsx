import React from 'react';
import './header.css';

function Header() {
  return (
    <div id="header">
      <p id="headertext">Content Types</p>
    </div>
  );
}

export default Header;
