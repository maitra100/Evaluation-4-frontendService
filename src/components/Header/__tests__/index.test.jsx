import renderer from 'react-test-renderer';
import React from 'react';
import Header from '../index';

jest.mock('react-router-dom');

describe('Login Section', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
