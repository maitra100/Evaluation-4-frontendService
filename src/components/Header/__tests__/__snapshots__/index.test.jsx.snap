// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Section renders correctly 1`] = `
<div
  id="header"
>
  <p
    id="headertext"
  />
</div>
`;
